scripts/globals/items.lua

```diff
@@ -342,4 +368,15 @@ xi.items =
     LIVING_BULLET                   = 21326,
     ERADICATING_BULLET              = 21327,
     MATRE_BELL                      = 21460,
+    PIEUJE_UNITY_SHIRT              = 25734,
+    AYAME_UNITY_SHIRT               = 25735,
+    IVINCIBLE_SHIELD_UNITY_SHIRT    = 25736,
+    APURURU_UNITY_SHIRT             = 25737,
+    MAAT_UNITY_SHIRT                = 25738,
+    ALDO_UNITY_SHIRT                = 25739,
+    JAKOH_WAHCONDALO_UNITY_SHIRT    = 25740,
+    NAJA_SALAHEEM_UNITY_SHIRT       = 25741,
+    FLAVIRIA_UNITY_SHIRT            = 25742,
+    YORAN-ORAN_UNITY_SHIRT          = 25743,
```

Comment:
Contrary to how we deal with filenames, I think in lua scripting we replace hypens with underscores